src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -261,6 +261,12 @@ def fuzz(self, target_path, options, reproducers_dir, max_time):
     # Check if we crashed, and get the crash testcase path.
     crash_testcase_file_path = runner.get_testcase_path(log_lines)
 
+    # If we exited with a non-zero return code with no crash file in output from
+    # libFuzzer, this is most likely a startup crash. Use an empty testcase to
+    # to store it as a crash.
+    if not crash_testcase_file_path and fuzz_result.return_code:
```

Comment:
It's probably safer to manually create an empty file. This seems like it could cause problems on windows.

Reply:
Good idea, forgot about windows. And this is safer anyway.

Reply:
Done!